src/app/contents/fields/it.ts

```diff
@@ -42,22 +42,27 @@ const it: Field[] = [
     title: "conforme",
     label: "Conforme",
     type: "object",
+    deprecated: true,
```

Comment:
I don't think this file is used anymore. The type SSOT should be `src/app/contents/publiccode.ts`.